tsconfig.base.json

```diff
@@ -33,6 +33,10 @@
       "utils-browser": [
```

Comment:
All Paths here should be removed!
Only @dlr-eoc and ol should stay.

Reply:
Done!